bait-files/src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -31,8 +31,27 @@
   private static final String AUTHENTICITY_RATING = "authenticity-rating";
   private static final String REVIEWER_NOTES = "reviewer-notes";
 
+  private class Result {
+      Request request;
+      Review review;
+
+      private Result(Review review, Request request){
+          this.review = review;
+          this.request = request;
+      }
+  }
+
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {}
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      try {
+      long reviewId = Long.parseLong(request.getParameter("reviewId"));
+      Review userReview = ObjectifyService.ofy().load().type(Review.class).id(reviewId).now();
+      Request userRequest =  ObjectifyService.ofy().load().type(Request.class).id(Long.parseLong(userReview.requestId)).now();
+
+      response.setContentType("application/json;");
+      response.getWriter().println(new Gson().toJson(new Result(userReview, userRequest)));
+    } catch (NumberFormatException e) { }    
```

Comment:
Do something as a result of this exception being caught... I think it would be weird to not send a response to the frontend, although I admit I don't really know what the most standard thing to do is, but maybe something like:
response.sendError(HttpServletResponse.SC_BAD_REQUEST);

Or some other error code on this page:
https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#SC_BAD_REQUEST